lib/active_campaign/tracking/event.ex

```diff
@@ -2,17 +2,28 @@ defmodule ActiveCampaign.Tracking.Event do
   @moduledoc """
   Documentation for `ActiveCampaign.Tracking.Event`.
   """
+  @event_url "https://trackcmp.net/event"
 
   alias ActiveCampaign.Http
+  alias ActiveCampaign.Config
 
-  # TODO endpoint is https://trackcmp.net/event ?
   # @doc """
   # Track event
   # """
-  # @spec track() :: {:ok, map()} | {:error, any()}
-  # def track() do
-  #   Http.post("")
-  # end
+  @spec track(String.t(), String.t(), map()) :: {:ok, map()} | {:error, any()}
+  def track(event, eventdata, visit) do
+    visit_json = Jason.encode!(visit)
```

Comment:
```suggestion
    visit_json = Config.json_library().encode!(visit)
```

Just like the http library, the jason library is configurable and up to the end user